<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>592</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>508</width>
    <height>592</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>508</width>
    <height>592</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sürücü Bilgilerini Güncelle</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
  background-color: #f2f6fc;
}

QGroupBox{
	background-color: #f2f6fc;
	border-radius: 15px;
	margin-top: 15px;
	font: 9pt &quot;Segoe UI&quot;;
	color: #5f5f5f
}

QTableWidget {
    border: 2px solid #e3ebf5;
    border-radius: 10px;
    background-color: #f2f6fc;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px 10px;
    background-color: #f2f6fc;
    border-radius: 7px;
    color: #5f5f5f;
}

QLabel{
	font: 9pt &quot;Segoe UI&quot;;
	color: #5f5f5f;
}

QLineEdit{
	background-color: transparent;
	border: 2px solid #ccd9e6;
	border-radius: 10px;
	font: 9 pt &quot;Segoe UI&quot; #5f5f5f;
}

QTextEdit{
	background-color: transparent;
	border: 2px solid #ccd9e6;
	border-radius: 10px;
	font: 9 pt &quot;Segoe UI&quot;;
}

QPushButton {
    background-color: #48B1DD;
    color: white;
    border: none;
    border-radius: 10px;
    padding: 6px 12px;
    font: 9pt &quot;Segoe UI&quot;;
}

QPushButton:hover {
    background-color: #007BFF;
}

QPushButton:pressed {
    background-color: #5F5F5F;
}
QComboBox {
    background-color: transparent;
    border: 2px solid #5f5f5f;
    border-radius: 10px;
    padding: 5px 8px;
    font: 9pt &quot;Segoe UI&quot;;
    color: #5f5f5f;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
    border-left: 1px solid #5f5f5f;
}


QComboBox QAbstractItemView {
    border: 2px solid #e3ebf5;
    selection-background-color: #e3ebf5;
    background-color: white;
    color: #5f5f5f;
    font: 9pt &quot;Segoe UI&quot;;
    border-radius: 5px;
}


</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupDriverAdd">
           <property name="title">
            <string>Sürücü Bilgilerini Güncelle</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>25</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QFormLayout" name="formLayoutAdd">
              <property name="horizontalSpacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="labelDriverTcNo">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>TC No</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditDriverTcNo">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelDriverName">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Adı</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditDriverName">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelDriverSurname">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Soyadı</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEditDriverSurname">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelDriverPhone">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Telefon</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEditDriverPhone">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="labelDriverAdress">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Adres</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QTextEdit" name="textEditDriverAdress">
                <property name="documentTitle">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayoutAddButton">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelDriverId">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonUpdateDriver">
                  <property name="minimumSize">
                   <size>
                    <width>90</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>90</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Güncelle</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
